File: hackerrank/the-love-letter-mystery/the-love-letter-mystery.py
#!/usr/bin/env python

numberOfCases = int(raw_input());

for n in xrange(numberOfCases):
	word = raw_input();
	operations = 0
	for i in xrange(len(word) / 2):
		operations = operations + abs(ord(word[i]) - ord(word[-(i+1)]))
	print operations

File: hackerrank/warmup/the-love-letter-mystery/the-love-letter-mystery.rb
#!/usr/bin/env ruby

numberOfTestCases = gets.to_i

(1..numberOfTestCases).each {
  word = gets.chomp
  operations = 0
  (0...word.size/2).each {|i|
    operations += (word[i].ord - word[-(i+1)].ord).abs
  }
  puts operations
}
